strings.txt

```diff
@@ -577,6 +577,8 @@ PLUGIN_QOBUZ_WORK
 
 PLUGIN_QOBUZ_RELEASE_TYPE
 	EN	Qobuz Release Type
+	FR	Type de sortie Qobuz
 
 PLUGIN_QOBUZ_RELEASES
 	EN	Releases
+	FR	Albums
```

Comment:
Thanks for this!  The translation of "Releases" as "Albums" doesn't seem consistent with others in this commit though. Shouldn't it be something like "Enregistrements" or "Sorties" to indicate that it includes Albums, EPs and Singles?

Reply:
I know... It doesn't seem coherent...
And, clearly, it's not perfect.

I've actually been annoyed with the translation of "release" since the feature appeared in LMS. To my knowledge (and according to friends I asked), there is no corresponding word in French. For a written work, we would speak of "publications", but for a musical work, there is nothing.

I asked myself the same questions when I had to translate this word in "slimserver" and in Material Skin. I varied the translations depending on the context of use. When it comes to choosing whether or not to activate the functionality (the classification of releases into albums, EPs, singles...), I try to favor "type de sortie". In most other cases, I use the generic word "albums", very rarely "enregistrements".

Strictly speaking, we could say that we are classifying "disques" (records) or "CD", but this is becoming “outdated”. ;-)

Here, for the Qobuz plugin, I tried to keep the same logic as elsewhere in LMS. I do not have a Qobuz account. It is difficult for me to be certain of the context of use.

I regularly follow forum posts in fear that someone will complain about my choices. So far, I haven't seen any negative reactions yet (but the feature is still very young and is not enabled by default in slimserver).

For Qobuz, in the end, maybe I should use "albums" for PLUGIN_QOBUZ_GROUP_RELEASES and "enregistrements" for PLUGIN_QOBUZ_GROUP_RELEASES_DESC. I let you choose.
````
PLUGIN_QOBUZ_GROUP_RELEASES
	DE	Gruppieren Sie Künstlerveröffentlichungen nach Typ
	FR	Regrouper les albums des artistes par type de sortie
````

*******

Dans slimserver :

```
SETUP_RELEASE_TYPES
	EN	Release types for Albums
	FR	Types de sorties pour les albums

SETUP_RELEASE_TYPES_INCLUDE
	EN	Release types to include in Albums list
	FR	Types de sorties à inclure dans la liste des albums

SETUP_RELEASE_TYPES_DESC
	EN	You can specify what release types you want to include in the albums list (see <a href="https://musicbrainz.org/doc/Release_Group/Type">MusicBrainz</a>). If a release doesn't have a type defined, "Album" is assumed. Albums can't be excluded from that list.
	FR	Vous pouvez spécifier quels types de sorties (voir <a href="https://musicbrainz.org/doc/Release_Group/Type">MusicBrainz</a>) doivent être inclus dans la liste des albums. Si aucun type n'a été défini pour une sortie, le type "Album" est pris par défaut. Le type "Album" ne peut pas être exclu.

SETUP_RELEASE_TYPES_COMPOSER_ALBUM_GROUP_OPTIONS
	EN	Composer album options
	FR	Regroupement des albums d'un compositeur

SETUP_RELEASE_TYPES_COMPOSER_ALBUM_GROUP_OPTIONS_DESC
	EN	Create a Composer Albums group - always, never, or for certain genres. If due to this setting, certain composer roles are not included in the Composer Albums group, they will appear in a separate "Compositions" group instead, which will list individual tracks, not albums. The genre list should be comma separated.
	FR	Créer un groupe "Albums du compositeur" : toujours, jamais ou pour des genres spécifiques. Si, en raison de ce paramètre, certaines œuvres du compositeur ne sont pas incluses dans le groupe "Albums du compositeur", elles apparaîtront à la place dans un groupe "Compositions" distinct, qui répertoriera les morceaux individuels, pas les albums. (Par exemple, un morceau avec un tag "ARTIST" différent du tag "COMPOSER" se retrouvera dans le groupe "Compositions".) Les genres de la liste doivent être séparés par des virgules.

SETUP_RELEASE_TYPES_COMPOSER_ROLE_GROUPING_0
	EN	Do not create a Composer Album group
	FR	Ne pas créer de groupe "Albums du compositeur"

SETUP_RELEASE_TYPES_COMPOSER_ROLE_GROUPING_1
	EN	Always create a Composer Album group
	FR	Créer toujours un groupe "Albums du compositeur"

SETUP_RELEASE_TYPES_COMPOSER_ROLE_GROUPING_2
	EN	Create a Composer Album group for these genres:
	FR	Créer un groupe "Albums du compositeur" pour ces genres :

SETUP_IGNORE_RELEASE_TYPES_0
	EN	Enable release type support for albums
	FR	Activer la prise en charge des types de sorties des albums

SETUP_IGNORE_RELEASE_TYPES_1
	EN	Ignore album release types
	FR	Ignorer les types de sorties des albums

SETUP_CLEANUP_RELEASE_TYPES
	EN	Try to recognize EPs and singles automatically when release type information is "album" or is missing.
	FR	Essayer de reconnaître automatiquement les EPs et les singles lorsque le type de sortie est manquant ou est de type "album".

SETUP_GROUP_BY_RELEASE_TYPES_0
	EN	Don't group releases
	FR	Ne pas regrouper les albums par type de sortie

SETUP_GROUP_BY_RELEASE_TYPES_1
	EN	Group an artist's lists by release type
	FR	Regrouper les albums d'un artiste par type de sortie

SETUP_GROUP_BY_RELEASE_TYPES_2
	EN	Group all lists by release type
	FR	Regrouper toutes les listes d'albums par type de sortie

ALL_RELEASES
	EN	All Releases
	FR	Tous les albums

RELEASETYPES_PROGRESS
	EN	Update Album release types
	FR	Actualiser les types de sorties des albums
````

*******

Dans Material Skin :

````
"...then album artist, release, disc no., track no.":"... puis artiste de l'album, album, n° de disque, n° de morceau",
"...then release, disc no., track no.":"... puis album, n° de disque, n° de morceau",
"Append random release to queue":"Ajouter un album aléatoire à la liste de lecture",
"Artist, Release":"Artiste, Album",
"Artist, Year, Release":"Artiste, Année, Album",
"Go to release":"Aller à l'album",
"Play release starting at track":"Jouer l'album à partir du morceau",
"Random Releases":"Albums aléatoires",
"Release":"Album",
"Release Mix":"Mélange d'albums",
"Release, Track Number":"Album, Numéro de Morceau",
"Releases":"Albums",
"Remove release":"Enlever l'album",
"Shuffle releases":"Mélanger les albums",
"Total releases: %1":"Nombre d'albums : %1",
"Use three lines (title, artist, release) to show track details.":"Utiliser trois lignes (titre, artiste, album) pour afficher les détails des morceaux.",
"Year, Artist, Release":"Année, Artiste, Album",
"Year, Release":"Année, Album",
"Year, Release, Track Number":"Année, Album, Numéro de Morceau",
"1 Release":"1 enregistrement",
"%1 Releases":"%1 enregistrements",
````

````
PLUGIN_MATERIAL_SKIN_RELEASE_TYPE_ORDER
	EN	Release type order
	FR	Ordre des types de sorties

PLUGIN_MATERIAL_SKIN_RELEASE_TYPE_ORDER_DESC
	EN	Specify order to display release types (this is a comma separated list of release type names) - e.g. "Album, EP, Single"
	FR	Spécifier l'ordre d'affichage des types de sorties (par une liste de types de sorties séparés par des virgules) - par ex. "Album, EP, Single"
````

Reply:
Now that I see the amount of thought and work you have put into this, I feel a little ashamed to have questioned your judgment on such a minor point. 😏
I defer to you on all your choices. Just let me know if you would like to make any further changes before this PR is merged. Thanks!

Reply:
I made the modification mentioned yesterday. The pull request can be merged.

No problem with the question that was asked yesterday. Quite the contrary! I prefer to be “challenged”. This can avoid mistakes.